modules/data-entry/src/main/frontend/src/questionnaireEditor/Question-hints.json

```
@@ -2,6 +2,9 @@
   "entryMode": "Specifies the source of the value for this question:  \n* manually entered by the **user**,\n* **computed** using a specified expression and answers to other questions, \n* copied over from another **reference** answer to a question specified by a path, or \n* **autocreated** by a backend script (the logic for populating autocreated answers cannot be configured from the user interface).",
   "expression": "Supports javascript syntax, with special placeholder variables being populated with answers to other questions in the form.\n\nExample:\n\n```return @{laps} * @{lapLength:-100}```\n\n where `laps` and `lapLength` are the names of questions in the same form, `:-` is an optional marker for a default value, and `100` is the optional default value.",
   "enableUserEntry": "If enabled, the user is not restricted to using one of the suggestions in the dropdown, and may save their own text as the free-form answer to the question.",
-  "question": "Path to a question, either in this or a different questionnaire. Example: `/Questionnaires/Demographics/name`"
+  "question": "Path to a question, either in this or a different questionnaire. Example: `/Questionnaires/Demographics/name`",
+  "defaultCountry": "The default country code from ISO 3166, ex. 'ca'",
+  "regions": "to show countries only from specified regions:  \n* ['america', 'europe', 'asia', 'oceania', 'africa'] \n* or subregions: \n* ['north-america', 'south-america', 'central-america', 'carribean', 'eu-union', 'ex-ussr', 'ex-yugos', 'baltic', 'middle-east', 'north-africa']",
+  "onlyCountries": "Array of country codes from ISO 3166 to be included in the select, ex. ['cu','cw','kz']"
```

We shouldn't have to ask users (even admins) to enter a well-formatted array with `[` and `'`. Instead, we should support entering a comma-separated list "us, ca". It would even greatly simplify the processing  of these settings in PhoneQuestion.

---

We already expext a comma separated string of countrries in `PhoneQuestion` so no simplification would follow.
Regarding the hint I'm not best at writing those so please help in formin it into the best wording.

---

I see now, we do expect the simple format "us, ca", it's just the hint that says otherwise.
```suggestion
  "regions": "Show countries only from specified regions  (e.g., `america, europe, asia, oceania, africa`)  or subregions (e.g. `north-america, south-america, central-america, carribean, eu-union, ex-ussr, ex-yugos, baltic, middle-east, north-africa`) in the country dropdown",
  "onlyCountries": "List of country codes to be made available in the dropdown, e.g. `cu, cw,kz`"
```